apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx

```diff
@@ -54,6 +53,10 @@ const ModuleNavigation: FC<Props> = ({
     if (external) servicePortalOutboundLink()
   }
 
+  useEffect(() => {
+    setExpand(isModuleActive)
+  }, [isModuleActive])
```

Comment:
setExpand should also be a dependency here.

Reply:
fixed